components/NewPersonView/Layout.tsx

```diff
@@ -136,6 +141,17 @@ const Layout = ({ person, children }: Props): React.ReactElement => {
         person={person}
       />
 
+      {case_note_deleted == true ? (
+        <AnnouncementMessage
+          title="Case note deleted"
+          content={`Case note "${String(
+            router.query.case_note_name
+          )}" has been deleted correctly`}
+        />
+      ) : (
+        <></>
+      )}
+
```

Comment:
This looks good, there is also a `confirmation banner` I'm not sure if this would be better to use? Although I'm happy with either